sdk/core/azure-core-amqp/inc/azure/core/amqp/internal/message_receiver.hpp

```diff
@@ -175,6 +175,15 @@ namespace Azure { namespace Core { namespace Amqp { namespace _internal {
      */
     std::pair<std::shared_ptr<const Models::AmqpMessage>, Models::_internal::AmqpError>
     TryWaitForIncomingMessage();
+#if _azure_TESTING_BUILD
+    // There is a deadlock associated with the link polling if it is enabled from an AMQP event
+    // callback. To work around this, link polling is disabled when creating a message receiver from
+    // an existing link endpoint. This method should be calledto enable it at a time when it is
```

Comment:
```suggestion
    // an existing link endpoint. This method should be called to enable it at a time when it is
```

Reply:
Please look at the most current iteration :).